back-end/src/sciler/communication/communication.go

```diff
@@ -19,17 +20,23 @@ type Communicator struct {
 // topicsOfInterest []string all topic to which to subscribe to
```

Comment:
please update documentation, this is probably relevant for most methods

Reply:
done